shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -311,7 +312,7 @@ public void onTouch(@NonNull PlatformViewsChannel.PlatformViewTouch touch) {
             vdControllers.get(touch.viewId).dispatchTouchEvent(event);
           } else if (platformViews.get(viewId) != null) {
             final MotionEvent event = toMotionEvent(density, touch, /*usingVirtualDiplays=*/ false);
-            View view = platformViews.get(touch.viewId);
+            View view = platformViews.get(touch.viewId).getView();
```

Comment:
nit:  should it check for `null` as well? Our API doesn't explicitly indicate that `getView()` is non-nullable.

Reply:
done